.gitignore

```diff
@@ -23,5 +24,4 @@ build
 *.idb
 *.exp
 
-example/addons/tiltfive/*.dll
```

Comment:
It's good practice not to include the end result of compiling the library in the output folder. Are we sure we want to remove this?

Reply:
I put the *.dll in the general extension section and then just white listed the libraries from Tilt Five